changelog/feature-IAN_client

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Not sure a feature is minor?

Reply:
I would say if its backwards compatible its minor. Its not a patch and i would consider major only changes breaking backwards compatibility.